druid/src/env.rs

```diff
@@ -69,6 +69,7 @@ pub enum Value {
     Color(Color),
     LinearGradient(Arc<LinearGradient>),
     Float(f64),
+    UnsignedFloat(u64),
```

Comment:
this seems a little odd. Maybe not meant to be committed? It doesn't seem to be used anywhere, and the internal type is not a float. It's possible we will want an `Integer` type here at some point (and maybe even an `UnsignedInteger`) but if we can get by with just `Float` for now that's great.

edit: okay, I see this is used for duration. so `UnsignedInt`?

Reply:
Ooooooh that is such a goof, I was under the impression that `u64` was an unsigned 64-bit float not an int :man_facepalming: I'll correct this and force push

Reply:
Annnnd corrected